nexus-tweaks.user.js

```diff
@@ -1144,6 +1144,264 @@ promiseList.push((async () => {
+        }
+
+        if (classifiedBadges.Career) {
+            const nodeL1 = L1("Career:", badgesNode)
+            Object.entries(orderObject(classifiedBadges.Career, Badges.Career)).forEach(e => {
+                const badge6 = ["Damage Dealt", "Damage Taken", "HP Healed"]
+                const nodeL2 = L2("\u2003" + e[0] + " [" + e[1].length + (badge6.includes(e[0]) ? "/6]" : "/5]"), nodeL1)
+                L3(e[1].join(", "), nodeL2)
+            })
+        }
+        if (classifiedBadges["Breath 5"]) {
+            const nodeL1 = L1("Breath 5:", badgesNode)
+            Object.entries(orderObject(classifiedBadges["Breath 5"], Badges["Breath 5"])).forEach(e => {
+                const badge6 = ["Damage Dealt", "Damage Taken", "HP Healed"]
+                const nodeL2 = L2("\u2003" + e[0] + " [" + e[1].length + "/2]", nodeL1)
+                L3(e[1].join(", "), nodeL2)
+            })
+        }
+        if (classifiedBadges["Breath 4"]) {
+            const nodeL1 = L1("Breath 4:", badgesNode)
+            Object.entries(orderObject(classifiedBadges["Breath 4"], Badges["Breath 4"])).forEach(e => {
+                const badge6 = ["Damage Dealt", "Damage Taken", "HP Healed"]
+                const nodeL2 = L2("\u2003" + e[0] + " [" + e[1].length + "/2]", nodeL1)
+                L3(e[1].join(", "), nodeL2)
+            })
+        }
+        if (classifiedBadges["Breath 3"]) {
+            const nodeL1 = L1("Breath 3:", badgesNode)
+            Object.entries(orderObject(classifiedBadges["Breath 3"], Badges["Breath 3"])).forEach(e => {
+                const badge6 = ["Damage Dealt", "Damage Taken", "HP Healed"]
+                const nodeL2 = L2("\u2003" + e[0] + " [" + e[1].length + "/2]", nodeL1)
+                L3(e[1].join(", "), nodeL2)
+            })
+        }
+        if (classifiedBadges.Exploration) {
+            const nodeL1 = L1("Exploration:", badgesNode)
+            Object.entries(orderObject(classifiedBadges.Exploration, Badges.Exploration)).forEach(e => {
+                const nodeL2 = L2("\u2003" + e[0] + " [" + e[1].length + "]", nodeL1)
+                L3(e[1].join(", "), nodeL2)
+            })
+        }
+        if (classifiedBadges["Monster Hunting"]) {
+            const nodeL1 = L1("Monster Hunting:", badgesNode)
+            Object.entries(orderObject(classifiedBadges["Monster Hunting"], Badges["Monster Hunting"])).forEach(e => {
+                const nodeL2 = L2("\u2003" + e[0] + " [" + e[1].length + "/3]", nodeL1)
+                L3(e[1].join(", "), nodeL2)
+            })
+        }
+        if (classifiedBadges.Others) {
+            const nodeL1 = L1("Others:", badgesNode)
+            L3(classifiedBadges.Others.join(", "), nodeL1)
+        }
+    }
+
+    const classify = (mod) => {
+        const badgesTitle = [...document.querySelectorAll("div.panetitle")].find(div => div.textContent == "Badges Earned:")
+        if (!badgesTitle) {
+            mod.log('Could not find badges pane')
```

Comment:
I think this is a debug level, not an info level event. This runs on every page load, so it logs when you're in the game.